containers/proxy-tftpd-image/tftp_wrapper.py

```diff
@@ -307,8 +358,9 @@ def main():
         exit(1)
 
     logging.info("Starting TFTP proxy:")
-    logging.info(f"httpHost: {args.httpHost}")
-    logging.info(f"proxyFqdn: {args.proxyFqdn}")
+    logging.info(f"forward host: {args.httpHost}")
```

Comment:
From log it would be hard to guess what it is for the person who did not implemented that. I assume it is _Hostname where to forward HTTP requests_ for help above. I would suggest using:
```suggestion
    logging.info(f"HTTP API Endpoint Host: {args.httpHost}")
```